src/effects.js

```diff
@@ -78,7 +78,6 @@ function createTween( value, prop, animation ) {
 }
 
 function defaultPrefilter( elem, props, opts ) {
-	/* jshint validthis: true */
```

Comment:
There would be 183 errors if we enable it, i'm not sure we want this, since we use seemingly contextless (just coined it!) functions all over, cause we love helpers for methods like `jQuery#forEach` and for them we have to use `call/bind` stuff